lib/ecl/ecl_grid.cpp

```diff
@@ -2685,9 +2686,9 @@ static ecl_grid_type * ecl_grid_alloc_GRDECL_kw__(ecl_grid_type * global_grid ,
                                                   const ecl_kw_type * gridhead_kw ,
                                                   const ecl_kw_type * zcorn_kw ,
                                                   const ecl_kw_type * coord_kw ,
-                                                  const ecl_kw_type * actnum_kw ,    /* Can be NULL */
+                                                  const int * actnum_data ,          /* Can be NULL */
```

Comment:
Please reorder the argument list here so that the `const int * actnum_data` comes either before all the `const ecl_kw_type *` arguments or after them - looks a bit funny when it comes in the middle.